storybook/stories/core/components/Input/QuickSearch.stories.js

```diff
@@ -1,21 +1,80 @@
-import {
-  BOTTOM,
-  DROPDOWN_POSITIONS,
-} from '@orangehrm/oxd/core/components/Input/types';
+import QuickSearchInput from '@orangehrm/oxd/core/components/Input/Autocomplete/QuickSearchInput';
+import QuickSearchInputReadOnly from './QuickSearchInput.readonly.story.vue';
+import QuickSearchInputDisabled from './QuickSearchInput.disabled.story.vue';
 import QuickSearchWidget from './QuickSearchInput.story.vue';
 
 export default {
   title: 'Inputs/QuickSearch',
-  component: QuickSearchWidget,
+  component: QuickSearchInput,
   argTypes: {
-    style: {control: {type: 'object'}},
-    hasError: {control: {type: 'boolean'}},
+    style: {
+      control: {type: 'object'},
+      table: {
+        type: {summary: 'Set custom style to the select'},
+      },
+    },
     dropdownPosition: {
-      options: DROPDOWN_POSITIONS,
-      defaultValue: BOTTOM,
+      options: ['top', 'bottom'],
+      control: {type: 'select'},
+      defaultValue: 'bottom',
+      table: {
+        type: {summary: 'Set drop down position for select'},
+      },
+    },
+    'dropdown:opened': {
+      control: {type: 'array'},
+      defaultValue: [],
+      table: {
+        type: {summary: 'emit event when dropdown opened'},
+      },
+    },
+    'dropdown:closed': {
+      control: {type: 'array'},
+      defaultValue: [],
+      table: {
+        type: {summary: 'emit event when dropdown closed'},
```

Comment:
First letter should be capital. Make sure other places also has the same

Reply:
updated